MesnardBarba-CiSE.md

```diff
@@ -118,26 +118,27 @@ Still, we ran against an obscure trick of the trade that changed our results com
-Despite the variations, the essence is the same, and it is reasonable to assume they would work similarly.
+Despite the variations, the two are essentially the same, and it is reasonable to assume they would work similarly.
 
 We already know that boundary conditions at the outlet of the computational domain can be problematic. This is no different with immersed boundary methods. 
 Our first attempt with IBAMR used a zero-gradient velocity boundary condition at the outlet. 
-This resulted in some blockage effect when the wake vortices reach the domain boundary: strong vorticity rebounds from the artificial boundary and propagates back to the domain (Figure 5). 
-Of course, this is unphysical and the result unacceptable. 
-After a long search in the literature and in the code documentation, we discovered that IBAMR needs us to select a "stabilized outlet," which is a boundary condition that acts like a force pushing the vortices out. 
+This results in a(?) blockage effect when the wake vortices reach the domain boundary: strong vorticity rebounds from the artificial boundary and propagates back to the domain (Figure 5). 
+Of course, this is unphysical and the result is unacceptable. 
+After a long search in the literature and in the code documentation, we discovered that IBAMR requires the selection of a "stabilized outlet," which is a boundary condition that acts like a force pushing the vortices out of the domain. 
 (IBAMR does not provide a convective/advective boundary condition.) 
 With this new configuration, the simulations of the snake profile resulted in a wake that looked physical, but a computed lift coefficient that was considerably different from our published study (Figure 6). 
-Another deep dive in the literature led us to notice that a benchmark example described in a paper describing extensions to IBAMR^(5) was set up in an unexpected way: 
+Another deep dive in the literature led us to notice that a benchmark example defined in a paper describing extensions to IBAMR^(5) was set up in an unexpected way: 
 the no-slip condition is forced _inside_ the body, and not just on the boundary. 
-As far as we could find, the publications using IBAMR are the only cases where interior points are constrained. 
+As far as we can(?) find, the publications using IBAMR are the only cases where interior points are constrained. 
 Other papers using immersed boundary methods apply the constraint only on boundary points.
-When we followed their example, our simulations with IBAMR were able to reproduce the lift enhancement at 35 degrees angle-of-attack, although with a slightly different value of average lift (<5% off). 
+When we followed their example, our simulations with IBAMR were able to reproduce the lift enhancement at angle-of-attack 35° (if consistent with previous formatting), although with a slightly different value of average lift (<5% off). 
```

Comment:
Yes, we should remove all instances of the character º, because it will not work on LaTeX.